cgotorch/cgotorch.cc

```diff
@@ -8,25 +8,25 @@
 Tensor RandN(int rows, int cols, int require_grad) {
   at::Tensor t = torch::randn({rows, cols},
                               at::TensorOptions().requires_grad(require_grad));
-  return new at::Tensor(std::move(t));
+  return new at::Tensor(t);
```

Comment:
There is only one data member in `at::Tensor`,  and it's a shared pointer. So, the copy constructor is enough?

Reply:
Yes, the copy-ctor and move-ctor are generated automatically, they're both viable and have little difference in performance.